EnglishFilter/sections/css_extended.txt

```diff
@@ -675,6 +675,9 @@ theverge.com##.l-col__main > div.c-compact-river > div.c-compact-river__entry[-e
 toolslib.net##[class="col-lg-4 col-md-6 col-sm-6"][-ext-has='.custom > div.custom-label > span:contains(Advertising)']
 ! https://forum.adguard.com/index.php?threads/23429/
 bhg.com.au##.OUTBRAIN ul.ob-widget-items-container > li.ob-dynamic-rec-container[-ext-has='a[onmousedown*="https://paid.outbrain.com/network/redir?"]']
+! https://github.com/AdguardTeam/AdguardFilters/issues/38725
+!+ PLATFORM(ext_ff, ext_opera)
+camwhores.tv#$?#body > div[style*="position"][style*="box-shadow"]:has(> a[href^="https://t.hrtyj.com/"]) { position: absolute!important; left: -3000px!important; }
```

Comment:
Not shown in code for me.

Reply:
better `camwhores.tv##body > div[style^="position: fixed; top: 50%; left: 50%; transform: translate(-50%, -50%); width: 60%; height: 60%;"]`

Reply:
<details><summary>Screenshot:</summary>

![1](https://user-images.githubusercontent.com/33488229/63584789-5acae480-c5a6-11e9-977e-a22cb28969e6.png)
</details><br/>